caddytls/storagetest/storagetest_test.go

```diff
@@ -0,0 +1,39 @@
+package storagetest
```

Comment:
I don't think we need to mock out the filesystem. Actually, in many of these cases use of the local FS is actually great integration/functional test anyways. Also note the existing tests, which I left intact as much as possible, use the FS. For example, I got Windows and Linux dir mod time testing for free when the CI ran.